src/components/calculator/calculator.ts

```diff
@@ -123,24 +123,42 @@ async function getQuote(billing: BillingClient, state: ICalculatorState): Promis
   const rangeStart = moment(state.rangeStart);
   const rangeStop = moment(state.rangeStop);
   const usageEvents = state.items.reduce((events: IUsageEvent[], item: IResourceItem) => {
```

Comment:
You told me to tell you to try changing this from a `reduce` to a `map`.

Reply:
Yeah, that is better. I've force pushed that update.